testdata/templates/example_cloud_run_mapping.json

```diff
@@ -0,0 +1 @@
+[{"name":"//run.googleapis.com/projects/{{.Provider.project}}/locations/us-central1/DomainMappings/tf-test-domain-meep.gcp.tfacc.hashicorptest.com","asset_type":"run.googleapis.com/DomainMapping","ancestry_path":"organization/529579013760/project/{{.Provider.project}}","resource":{"version":"run.googleapis.com","discovery_document_uri":"https://www.googleapis.com/discovery/v1/apis/run/run.googleapis.com/rest","discovery_name":"DomainMapping","parent":"//cloudresourcemanager.googleapis.com/projects/{{.Provider.project}}","data":{"apiVersion":"domains.cloudrun.com/v1","kind":"DomainMapping","metadata":{"name":"tf-test-domain-meep.gcp.tfacc.hashicorptest.com","namespace":"{{.Provider.project}}"},"spec":{"certificateMode":"AUTOMATIC","routeName":"tf-test-cloudrun-srv-beep"}}}},{"name":"//run.googleapis.com/projects/{{.Provider.project}}/locations/us-central1/services/tf-test-cloudrun-srv-beep","asset_type":"run.googleapis.com/Service","ancestry_path":"organization/529579013760/project/{{.Provider.project}}","resource":{"version":"run.googleapis.com","discovery_document_uri":"https://www.googleapis.com/discovery/v1/apis/run/run.googleapis.com/rest","discovery_name":"Service","parent":"//cloudresourcemanager.googleapis.com/projects/{{.Provider.project}}","data":{"apiVersion":"serving.knative.dev/v1","kind":"Service","metadata":{"name":"tf-test-cloudrun-srv-beep","namespace":"{{.Provider.project}}"},"spec":{"template":{"spec":{"containers":[{"image":"us-docker.pkg.dev/cloudrun/container/hello"}]}}}}}}]
```

Comment:
it might be nice to pretty-print the JSON files for easier human consumption (for example by running them through [jq](https://stedolan.github.io/jq/))